### DISCUSSION TITLE
Header

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#246] [RELEASED: Components, pages and patterns]
# Existing

- [Header](https://design-patterns.service.justice.gov.uk/components/header/) on the MOJ Pattern Library

# Variation 

We need to add a variation for when:

- we want to display the phase banner to remind users that it's not a live service yet.
- we don't want to display a feedback banner, for example an internal service will have different feedback loops (e.g. dedicated support channel on Teams).

![Screenshot 2021-08-18 at 15 04 32](https://user-images.githubusercontent.com/6122118/129912579-3b98d584-081a-47e3-9cf3-2283086c2da5.png)

**Comment by JSJohal** [comment]
**Digital Prison Services - internal staff service header**

![image](https://user-images.githubusercontent.com/22663194/130777929-320c0dd6-260e-456f-bd55-7171deba391a.png)

This is the variation that is used across Digital Prison Services for internal staff facing services (prison officer services).

We've got a few distinct differences from the pattern library header:

- We've removed the blue bar underneath the header, previous DPS user research found that the blue bar resulted in pushing content down the page and provided no value to users, this was especially prominent when factoring in the screen size of the legacy devices used by prison staff
- We've added a 'manage your details' link which links to the user account in HMPPS Auth, this is where users change their password / email etc
- The name of the account holder is tweaked also, the first name is initialised, such as J. Smith: rationale behind this is that prison officers have expressed before that as some of these services will be accessed with prisoners in their vicinity, they would not want the prisoner to know their first name.

**Comment by lalexander12** [comment]
There's a line in the guidance that says:

> Use the MOJ header component for any service or system not on GOV.UK like internal staff.

Do you mean that the MOJ header should be used for internal staff facing services?

What about moj services that are for restricted external users? (For example, legal aid solicitors who sign in to use our services)

A lot of moj services are on a url which ends service.justice.gov.uk - Does that mean that they are **not** on gov.uk?

**Comment by lalexander12** [comment]
On our Legal Aid Agency services we've combined the MOJ header with the gov service navigation

<img width="745" height="136" alt="Screenshot 2026-02-05 at 10 01 06" src="https://github.com/user-attachments/assets/0de0636f-7b5a-49a5-bab9-2205b13b6baf" />

<br>
<br>


<img width="679" height="83" alt="Screenshot 2026-02-05 at 10 01 12" src="https://github.com/user-attachments/assets/710aba1b-8ae2-4e31-9be4-169dd01f3681" />

**Comment by phurst-gh** [comment]
Hi there, Im not sure of the rules around who is allow to commit to this branch, but I spotted somethign that looks out of place to me (could be wrong).

The header has this line:
`<a class="moj-header__navigation-link" href="#" aria-current="page">Account name</a>`

I believe `aria-current="page"` indicates that the element using this attribute represents the current page. Where an account name does not.

Im happy to add this/post PR if Im allowed and it helps.
